### PR TITLE
Minor grammatical tweaks to README file

### PR DESCRIPTION
**Pull request by cristofolini**
As I was reading our README I came across some grammatical mistakes and typos, so I've taken the liberty to fix whatever I could find so as to make the text more gramatically coherent/correct.

Little mistakes like these are bound to happen since none of us are native speakers, so I suppose the best we can do is help each other :)

I'm open to suggestions and questions regarding any of the changes, of course.

### CODE REVIEW

**Comment by rafaelweingartner** [comment]
@cristofolini those are great suggestions.
It LGTM

**Comment by rafaelweingartner** [comment]
ok to test

File: README.md [hunk 1]
```diff
@@ -2,23 +2,23 @@
...
 
-Our mission is to take the orchestration of cloud computing infrastructures to the next level. To achieve that, we developed a distributed and autonomic plugin that can monitor the CC environment and take decisions in order to fulfill the goals of the given environment. We do not wish to remove the need for human administrators, but to improve their work. Our mission is to optimize and make CC environments more efficient and stable without needing to rely on human administrators.
+Our mission is to take the orchestration of cloud computing infrastructures to the next level. To achieve that, we've developed a distributed and autonomic plugin that can monitor a CC environment and take decisions in order to fulfill the goals of said environment. We do not wish to remove the need for human administrators, but to improve their work. Our mission involves optimizing and making CC environments more efficient and stable without the need to rely on human administrators.
```
Comment (rafaelweingartner) [inline README.md]: @cristofolini here you use a contraction:

> we've

but in writing I think we should avoid them. Could you use the full version of the verb?

**Comment by GabrielBrascher** [comment]
@cristofolini good improvements, it LGTM.

If you don't mind, we would appreciate a review on our website also (http://autonomiccs.com.br/website/ - https://github.com/Autonomiccs/autonomiccs-website/blob/master/index.html), thanks ;)

**Comment by cristofolini** [comment]
@rafaelweingartner That's a fair point. I've changed all contractions on the text to their normal forms.
@GabrielBrascher Will do!

**Pull request merged**